From 71ad791bb531272fa3272b962dd1e58b2e4ad789 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Tue, 26 May 2020 20:55:00 +0300
Subject: [PATCH 01/99] ucm-imx8m-mini: update CPU/model printed parameters

Update the startup messages of the CPU and model parameters

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm/dts/ucm-imx8m-mini.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/ucm-imx8m-mini.dts b/arch/arm/dts/ucm-imx8m-mini.dts
index 112345b6a3..899899c1c6 100644
--- a/arch/arm/dts/ucm-imx8m-mini.dts
+++ b/arch/arm/dts/ucm-imx8m-mini.dts
@@ -17,7 +17,7 @@
 #include "fsl-imx8mm.dtsi"
 
 / {
-	model = "CompuLab UCM i.MX8MM board";
+	model = "CompuLab UCM-iMX8M-Mini board";
 	compatible = "cpl,ucm-imx8m-mini", "fsl,imx8mm-evk", "fsl,imx8mm";
 
 	chosen {
-- 
2.11.0

